Formula/sqlite-analyzer.rb

```diff
@@ -13,7 +13,7 @@ class SqliteAnalyzer < Formula
   end
 
   def install
-    system "./configure", "--disable-debug", "--prefix=#{prefix}"
+    system "./configure", "--disable-debug", "--with-tcl=/System/Library/Frameworks/Tcl.framework/", "--prefix=#{prefix}"
```

Comment:
Should be `--with-tch=#{MacOS.sdk_path}/System/Library/Frameworks/Tcl.framework`

Reply:
Yes, it's good.